TM1py/Services/ServerService.py

```diff
@@ -205,30 +205,36 @@ def utc_localize_time(timestamp):
     @require_admin
     def get_transaction_log_entries(self, reverse: bool = True, user: str = None, cube: str = None,
                                     since: datetime = None, until: datetime = None, top: int = None,
-                                    elements: Dict[str, str] = None, **kwargs) -> Dict:
+                                    element_tuple_filter: Dict[str, str] = None,
+                                    element_position_filter: Dict[int, Dict[str, str]] = None, **kwargs) -> Dict:
         """
         :param reverse: Boolean
         :param user: UserName
         :param cube: CubeName
         :param since: of type datetime. If it doesn't have tz information, UTC is assumed.
         :param until: of type datetime. If it doesn't have tz information, UTC is assumed.
         :param top: int
-        :param elements: of type dict. Element name as key and comparison operator as value
+        :param element_tuple_filter: of type dict. Element name as key and comparison operator as value
+        :param element_position_filter: not yet implemented
         tuple={'Actual':'eq','2020': 'ge'}
         :return:
         """
+        if element_position_filter:
+            raise NotImplementedError("Feature expected in upcoming releases of TM1, TM1py")
+
         reverse = 'desc' if reverse else 'asc'
         url = '/api/v1/TransactionLogEntries?$orderby=TimeStamp {} '.format(reverse)
+
         # filter on user, cube and time
         if user or cube or since or until:
```

Comment:
I think `or element_tuple_filter or element_position_filter` should be added here

Reply:
good catch. Thank you